e2e/_suites/kubernetes-autodiscover/testdata/kind.yml

```diff
@@ -0,0 +1,4 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
```

Comment:
>[2021-04-22T18:44:05.639Z] time="2021-04-22T18:44:04Z" level=error msg="Error executing command" args="[create cluster --name kind-590267e1-989e-49dd-ab45-e97edaf4a313 --config testdata/kind.yml --kubeconfig /tmp/test-335526962/kubeconfig]" baseDir=. command=kind error="exit status 1" stderr="Creating cluster \"kind-590267e1-989e-49dd-ab45-e97edaf4a313\" ...\nERROR: failed to create cluster: unable to decode config: yaml: unmarshal errors:\n  line 3: field featureGates not found in type v1alpha4.Cluster\n"

This error is raised on CI

Reply:
Umm, this works for me:tm: with kind 0.10.0, what version of kind is used in CI?

Reply:
https://github.com/elastic/e2e-testing/blob/249fd97a14b97fe7e51731651b2335b57df1da33/.ci/Jenkinsfile#L53

0.10.0

Reply:
Ok, it seems that each suite has its own script to install dependencies, I was not providing one and the machine had kind 0.7.0 installed. Added a script to install the expected versions of kind and kubectl.